test/metabase/test/data/interface.clj

```diff
@@ -163,6 +163,15 @@
   [driver]
   (test-runner.init/assert-tests-are-not-initializing (pr-str (list 'the-driver-with-test-extensions driver)))
   (initialize/initialize-if-needed! :plugins)
+  ;; without this check, people will see an error like the following
+  ;;   Caused by: java.io.FileNotFoundException: Could not locate metabase/driver/all__init.class,
+  ;;   metabase/driver/all.clj or metabase/driver/all.cljc on classpath.
+  ;; add a more explicit check to make it clear what the problem is
+  (when (= :all driver)
```

Comment:
I'm confused by this addition. I get the idea of it, but was it a problem? If people start with `DRIVERS=foo` they would have the same problem. Is there a reason that `DRIVERS=all` is an expected allowed option?

Reply:
ha, this is Jeff's thing. I thought we have a way to set a magic keyword or sth but didn't really look into it. 

Did a quick grep and seems like we don't, so imma removes this change.